src/dialects/oracle/query/compiler.js

```diff
@@ -105,9 +105,19 @@ assign(QueryCompiler_Oracle.prototype, {
   update: function() {
     var updates = this._prepUpdate(this.single.update);
     var where   = this.where();
-    return 'update ' + this.tableName +
-      ' set ' + updates.join(', ') +
-      (where ? ' ' + where : '');
+    var returning = this.single.returning;
+
+    if (!returning) {
+      return 'update ' + this.tableName + ' set ' + updates.join(', ') + (where ? ' ' + where : '');
```

Comment:
@wubzz Amended once more with line breaks in the sql statement to pass all the travis build tests